### PR TITLE
cleanup unused proto

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
1) remove `read_quorum` for `Context`;
2) remove `binlog` for `CommitRequest`.

### CODE REVIEW

**Comment by disksing** [comment]
LGTM

File: proto/kvrpcpb.proto [hunk 1]
```diff
@@ -34,7 +34,6 @@ message Context {
     uint64 region_id = 1;
     metapb.RegionEpoch region_epoch = 2;
     metapb.Peer peer = 3;
-    bool read_quorum  = 4;
```
Comment (BusyJay) [inline proto/kvrpcpb.proto]: Please mark it as reserved field.

File: proto/kvrpcpb.proto [hunk 2]
```diff
@@ -102,7 +102,6 @@ message CommitRequest {
     uint64 start_version = 2;
     repeated bytes keys = 3;
     uint64 commit_version = 4;
-    bytes  binlog = 5;
```
Comment (siddontang) [inline proto/kvrpcpb.proto]: reserved ditto?

**Comment by siddontang** [comment]
LGTM
PTAL @BusyJay

**Comment by BusyJay** [comment]
LGTM, though I'm worry about how to trigger `read_quorum` in the future. And several tests will be broken for sure.

**Pull request merged**